frontend/src/app.css

```
@@ -433,7 +433,7 @@
   --border-faint: #f0f0f0;
   --border: #e5e5e5;
   --border-strong: #c4c4c4;
-  --focus-ring: rgba(17, 17, 17, 0.10);
+  --focus-ring: rgba(17, 17, 17, 0.1);
```

Stylelint (17.14.0)

[error] Duplicate custom property "--focus-ring" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)